CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ endif()
 project(Orbit C CXX)
 enable_testing()
 
+option(WITH_UI "Setting this option will enable the Qt-based UI client." ON)
```

Comment:
Nit: GUI instead of UI?

Reply:
Yes, sounds better. I'm gonna change that. Hold on...